web-admin-app/src/main/java/edu/unc/lib/boxc/web/admin/controllers/ChompbController.java

```diff
@@ -48,16 +51,45 @@ public String chompbCroppingReport() {
         return "report/chompb";
     }
 
-    @RequestMapping(value = "chompb/project/{projectName}/processing_results/{jobName}/data.json",
-            method = RequestMethod.GET,
-            produces = APPLICATION_JSON_VALUE)
-    public @ResponseBody String getProcessingResults(@PathVariable("projectName") String projectName,
-                                                     @PathVariable("jobName") String jobName) {
+    /**
+     * Get processing result files for a specific job
+     * @param projectName
+     * @param jobName
+     * @param filePath
+     * @return
+     * @throws IOException
+     */
+    @RequestMapping(value = "chompb/project/{projectName}/processing_results/{jobName}/files",
+            method = RequestMethod.GET)
+    public ResponseEntity<InputStreamResource> getProcessingResults(@PathVariable("projectName") String projectName,
+                                                @PathVariable("jobName") String jobName,
+                                                @RequestParam(value = "path", defaultValue = "false") String filename)
+                                                throws IOException {
+//        var filename = extractFilename(request, jobName);
         var agentPrincipals = AgentPrincipalsImpl.createFromThread();
-        try {
-            return chompbPreIngestService.getProcessingResults(agentPrincipals, projectName, jobName);
-        } catch (IOException e) {
-            throw new RuntimeException(e);
+//        filename = URLDecoder.decode(filename, StandardCharsets.UTF_8);
```

Comment:
Should this be removed?

Reply:
I think this got removed already if you look at the final state of the PR